libkineto/src/RoctracerActivity.h

```diff
@@ -101,6 +101,15 @@ struct GpuActivity : public RoctracerActivity<roctracerAsyncRow> {
   void log(ActivityLogger& logger) const override;
   const std::string metadataJson() const override;
 
+  // Add small buffer to fix visual error created by https://github.com/ROCm/roctracer/issues/105
+  // Once this is resolved we can use ifdef to handle having this buffer or not based on version
+  int64_t timestamp() const override {
+    return activity_.begin + 1;
+  }
+  int64_t duration() const override {
+    return activity_.end - (activity_.begin + 1);
```

Comment:
should it be -2 , so that it finishes right before? Or is it okay if end time == parent end time?

Reply:
So we are only shaving off a nanosecond off the beginning since this is on the gpu track only so there won't be any parent/child minus the annotations which isn't a big deal since we set those ourselves.